Scenario14_AggressiveWatcherLifecycle.cs

```diff
@@ -0,0 +1,169 @@
+namespace FswStressTests;
+
+/// <summary>
+/// Scenario 14: Aggressive watcher lifecycle stress test
+/// Tests edge cases in watcher lifecycle management:
+/// - Multiple threads rapidly creating and destroying watchers on the same directory
+/// - File operations happening during watcher creation and disposal
+/// - Tests disposal during active event processing
+/// - Verifies global watcher handles rapid watch additions/removals
+/// </summary>
+public class Scenario14_AggressiveWatcherLifecycle : IStressScenario
+{
+    public string Name => "Aggressive Watcher Lifecycle Management";
+
+    public async Task RunAsync()
+    {
+        string testDir = Path.Combine(Path.GetTempPath(), $"fsw_test_aggressive_lifecycle_{Guid.NewGuid()}");
+        Directory.CreateDirectory(testDir);
+
+        try
+        {
+            int threadCount = Environment.ProcessorCount;
+            int iterationsPerThread = 50;
+            System.Collections.Concurrent.ConcurrentBag<Exception> exceptions = new();
```

Comment:
Don't use Fully qualified names.

```suggestion
            ConcurrentBag<Exception> exceptions = new();
```